addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -105,6 +105,9 @@ FormRenderer.include({
      * @returns {jQuery.Element}
      */
     _makeChatterContainerTarget() {
+        if (this._chatterContainerTarget) {
+            return $(this._chatterContainerTarget);
+        }
```

Comment:
Is the diff in this file possible thanks to change in this PR or in Owl 2?
Otherwise it should be done in earlier version because we have issues due to multiple renders in a row that this might fix. Or at least in a separate PR to be able to test it alone.

Reply:
It's because there are limitations in the compatibility layer with owl2, in particular, it does support that the parent element of a Component changes. So the purpose of this diff is simply to keep the same element.